exoscale/resource_exoscale_database_kafka_test.go

```diff
@@ -14,7 +14,7 @@ import (
 
 var (
 	testAccResourceDatabaseKafkaIPFilter = []string{"1.2.3.4/32"}
-	testAccResourceDatabaseKafkaVersion  = "3.1"
+	testAccResourceDatabaseKafkaVersion  = "3.2"
 	testAccResourceDatabasePlanKafka     = "business-4"
```

Comment:
It could be nice if we can fix this value durably (maybe to be handled in a separate story).

Reply:
I agree. Same applies for other databases so I'll create a separate story.

Reply:
Story created.